File: src/main/java/hantonik/atomic/core/item/IBurnableItem.java
package hantonik.atomic.core.item;

import net.minecraft.world.item.ItemStack;
import net.minecraft.world.item.crafting.RecipeType;
import net.minecraftforge.common.extensions.IForgeItem;
import org.jetbrains.annotations.Nullable;

public interface IBurnableItem extends IForgeItem {
    @Override
    int getBurnTime(ItemStack itemStack, @Nullable RecipeType<?> recipeType);
}

File: Forge/src/main/kotlin/hantonik/atomic/core/item/IBurnableItem.kt
package hantonik.atomic.core.item

import net.minecraft.world.item.ItemStack
import net.minecraft.world.item.crafting.RecipeType
import net.minecraftforge.common.extensions.IForgeItem

interface IBurnableItem : IForgeItem {
    override fun getBurnTime(itemStack: ItemStack, recipeType: RecipeType<*>?): Int
}